tests/jvm/src/test/resources/metac-metacp.expect.diff

```diff
@@ -3,9 +3,8 @@ advanced.D#
 ===========
 --- metac
 +++ metacp
-                 type_arguments {
-                   tag: TYPE_REF
                    typeRef {
+                     prefix {
```

Comment:
What does `prefix {` do here?

Reply:
That's a good question. It seems caused by the removal of `Option[T]` wrapping. Currently, we're producing roughly the equivalent of `TypeRef(Some(Type.defaultMessage), ...)` which is not the same as `TypeRef(None, ...`. I think we need to change the behavior of the no_box option to guard against the case when `Some(defaultValue) == None`, seems fixable but unrelated to the sealed oneof PR. It should likely be a PR changing the `(scalapb.field).no_box` option.

Reply:
Please open a ticket for this. Otherwise LGTM.

Reply:
Done. https://github.com/scalameta/scalameta/issues/1607